examples/docs/en-US/dropdown.md

```diff
@@ -104,6 +104,29 @@ Click the triggering element or hover on it.
 </el-row>
 ```
 :::
+
+### The way of hiding menu
+
+use `hide-on-click` to change the way of hiding menu.
+
+:::demo The default is to close the menu when you click on menu items, this feature can be turned off by setting `hide-on-click` to false.
```

Comment:
By default menu will close when you click on menu items, and it can be turned off by setting `hide-on-click` to false.